plugin/src/main/java/org/opensearch/ml/action/undeploy/TransportUndeployModelAction.java

```diff
@@ -246,7 +245,6 @@ private MLUndeployModelNodeResponse createUndeployModelNodeResponse(MLUndeployMo
         }
 
         Map<String, String> modelUndeployStatus = mlModelManager.undeployModel(modelIds);
-        mlStats.getStat(MLNodeLevelStat.ML_EXECUTING_TASK_COUNT).decrement();
```

Comment:
What will happen after the request done? Do we still show there is one request executing ?